playbooks/example-use/playbook.yml

```diff
@@ -9,6 +9,6 @@
         msg: >
           "You must update Ansible to at least 2.8 to use this version of Pulp 3 Installer."
   roles:
-    - pulp_all_services
+    - pulp.pulp_installer.pulp_all_services
```

Comment:
How about we specify `collections:` at the top of the playbook instead of the fully qualified collection names?
https://docs.ansible.com/ansible/latest/user_guide/collections_using.html#using-collections-in-a-playbook

Reply:
This is a nice feature, yes. But introduced in ansible 2.9.

I feel like i have stated before, that ansible 2.8 is really bad with collections...

Reply:
Also to my knowledge this `collections:` feature only works for roles and modules, not for filter and lookup and other plugins.

Reply:
https://github.com/ansible/ansible/issues/68198